routes/api/assets/index.js

```diff
@@ -79,7 +79,8 @@ router.get('/:asset_id', (req, res, next) => {
     .findById(req.params.asset_id)
     .then((asset) => {
       if (!asset) {
-        return res.status(404).end();
+        res.status(404).end();
+        return;
```

Comment:
It wasn't documented what `res.status(404).end();`, so I didn't feel it necessary for it to return it's result through the promise chain.